docs/content/tutorial/setup.md

```diff
@@ -0,0 +1,74 @@
+# Setup the environment
```

Comment:
What about nudging to use multipass, like [here](https://charmhub.io/postgresql/docs/t-set-up#heading--multipass)? This would be cross-environemt (multipass works on Mac, Windows too), and already comes with all the tooling pre-installed. 

I know that in Kafka we are suggesting to setup from scratch, and it may not be worth re-wording, but the multipass seems a better option that we could nudge to when creating new docs/tutorial